opm/output/data/Solution.cpp

```diff
@@ -64,6 +64,20 @@ Solution::insert(std::string               name,
 }
 
 
+std::pair<Solution::iterator, bool>
+Solution::insert(std::string               name,
+                 const UnitSystem::measure m,
+                 std::vector<float>        xs,
+                 const TargetType          type)
+{
+    std::vector<double> xsd(xs.size());
+    std::copy(xs.begin(), xs.end(), xsd.begin());
```

Comment:
You can probably write this as
```C++
std::vector<double> xsd(xs.begin(), xs.end());
```
in order to avoid construct+copy here.

Reply:
done